### ISSUE TITLE
Please put eval $(thefuck --alias) in your ~/.zshrc.

### ISSUE THREAD

**Issue by raeesi** [nvbn/thefuck#585]
I've just installed thefuck by  sudo -H pip install thefuck & now when i type fuck and hit the enter it says:

```
Seems like fuck alias isn't configured!
Please put eval $(thefuck --alias) in your ~/.zshrc.
More details - https://github.com/nvbn/thefuck#manual-installation

```
when i added the line to my .zsh file, it's still same.
what should i do now to fix the problem?

**Comment by mfwass** [comment]
A similar issue occurs on macOS Sierra.

```
Michaels-MacBook-Pro:base Michael$ fuck
Seems like fuck alias isn't configured!
Please put eval $(thefuck --alias) in your ~/.bashrc.
More details - https://github.com/nvbn/thefuck#manual-installation
```

It _is_ already in my `~/.bashrc`.

![image](https://cloud.githubusercontent.com/assets/8769835/21960176/a0c4e75a-daae-11e6-8dbf-068464a6c020.png)

**Comment by nvbn** [comment]
Have you tried restarting shell or doing `source ~/.bashrc`?

**Comment by mfwass** [comment]
@nvbn 
```
Michaels-MacBook-Pro:~ Michael$ source ~/.bashrc
Michaels-MacBook-Pro:~ Michael$ fuck
open ~/.bashrc [enter/↑/↓/ctrl+c]
Aborted
```

Restarted terminal ->

```
Michaels-MacBook-Pro:~ Michael$ fuck
Seems like fuck alias isn't configured!
Please put eval $(thefuck --alias) in your ~/.bashrc.
More details - https://github.com/nvbn/thefuck#manual-installation
```

**Comment by rnestler** [comment]
> It appears to work until I need to restart Terminal.

So probably on OSX `~/.bashrc` doesn't get sourced automatically? [This](https://superuser.com/questions/147043/where-to-find-the-bashrc-file-on-mac-os-x-snow-leopard-and-lion#147699) stackoverflow question may give some insight.

**Comment by mfwass** [comment]
@rnestler This works, however it will open `.bash_profile` every time I invoke `fuck` for the first time.

**Comment by nvbn** [comment]
There's a bug, it should suggest `.bash_profile` if `.bashrc` doesn't exists.

**Comment by graciano** [comment]
I just started using mac and I had no ideia about this. This thread helped a lot. Thanks :)

**Comment by mfwass** [comment]
Over time the issue with TextEdit popping up went away.  Looks like it was
fetching it from a prior command I had made.  Adding the `eval....` to
`.bash_profile` solves the issue entirely.

**Comment by mfwass** [comment]
Over time the issue with TextEdit popping up went away.  Looks like it was
fetching it from a prior command I had made.  Adding the ```eval....``` to
```.bash_profile``` solves the issue entirely.

**Comment by nvbn** [comment]
Fixed in 3.16.

**Comment by mfwass** [comment]
💯
